<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMoneyIO</class>
 <widget class="QWidget" name="WidgetMoneyIO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>37</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>261</width>
    <height>37</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>261</width>
    <height>37</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_value">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>101</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(52, 53, 56);
	border-top: 1px  solid rgb(150, 150, 150) ;
	border-left: 0px  solid rgb(150, 150, 150) ;
	border-bottom: 1px  solid rgb(150, 150, 150) ;
	border-right: 1px  solid rgb(150, 150, 150) ;
}</string>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>31</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 0px;
	background-color: rgb(52, 53, 56);
	border-top: 1px  solid rgb(150, 150, 150) ;
	border-left: 1px  solid rgb(150, 150, 150) ;
	border-bottom: 1px  solid rgb(150, 150, 150) ;
	border-right: 1px  solid rgb(150, 150, 150) ;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(52, 53, 56);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_unit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>31</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
	background-color: rgb(52, 53, 56);
	border-top: 1px  solid rgb(150, 150, 150) ;
	border-left: 0px  solid rgb(150, 150, 150) ;
	border-bottom: 1px  solid rgb(150, 150, 150) ;
	border-right: 1px  solid rgb(150, 150, 150) ;
</string>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
